ext/crates/fp/src/limb.rs

```diff
@@ -11,6 +10,148 @@ pub(crate) struct LimbBitIndexPair {
     pub(crate) bit_index: usize,
 }
 
+/// A struct that defines a range of entries in a slice of limbs. We number limbs starting from some
+/// other limb that might not be in the range. This is because we must be able to index a struct
+/// using indices for another ambient struct. For example, accessing a `Slice` using coordinates
+/// from an ambient `FpVector`.
+#[derive(Debug, Hash, PartialEq, Eq, Clone, Copy)]
+pub struct LimbLength<const P: u32> {
```

Comment:
This is bikeshedding territory but I would say "LimbRange" is a better name.

Reply:
I went through a few iterations with the naming. I initially went with `ColumnSize` since we'll be reusing that struct to define the columns of our matrices, but `ColumnRange`, `LimbRange`, etc. are all good options I think